sunbeam_migrate/tests/integration/handlers/neutron/test_floating_ip.py

```diff
@@ -0,0 +1,206 @@
+# SPDX-FileCopyrightText: 2025 - Canonical Ltd
+# SPDX-License-Identifier: Apache-2.0
+
+from sunbeam_migrate.tests.integration import utils as test_utils
+
+
+def _create_external_network(session):
```

Comment:
Can we reuse those helpers and add parameters if needed?

https://github.com/petrutlucian94/sunbeam-migrate/blob/main/sunbeam_migrate/tests/integration/handlers/neutron/utils.py